frontend/src/pages/TaskOverviewPage.tsx

```diff
@@ -20,10 +20,21 @@ export const TaskOverviewPage = () => {
   }>();
   const history = useHistory();
   const task = useSelector(taskSelector(Number(taskId)));
+  const tasks = useSelector(allTasksSelector(), shallowEqual);
+  const taskIdx = tasks.findIndex(({ id }) => Number(taskId) === id);
 
   const siblingTasks = [
-    { id: useSelector(taskSelector(Number(taskId) - 1))?.id, type: 'previous' },
-    { id: useSelector(taskSelector(Number(taskId) + 1))?.id, type: 'next' },
+    {
+      id: taskIdx >= 0 && taskIdx - 1 >= 0 ? tasks[taskIdx - 1].id : undefined,
+      type: 'previous',
+    },
+    {
+      id:
+        taskIdx >= 0 && taskIdx + 1 <= tasks.length - 1
```

Comment:
```suggestion
        taskIdx >= 0 && taskIdx + 1 < tasks.length
```
`idx < length` would be more idiomatic for limit check